code-security/admin_guide/scan-monitor/secrets-scanning.adoc

```diff
@@ -0,0 +1,52 @@
+== Secrets Scanning
+
+You can use Code Security to detect and remove secrets in IaC files stored in your IDEs, Git-based VCS, and CI/CD pipelines.
```

Comment:
We can't "remove" secrets, so should we say "block"?

Reply:
done